app/views/claims/qts_year.html.erb

```diff
@@ -1,26 +1,28 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: {qts_award_year: "tslr_claim_qts_award_year_before_2013"}) if current_claim.errors.any? %>
+    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: {:"eligibility.qts_award_year" => "tslr_claim_eligibility_attributes_qts_award_year_before_2013"}) if current_claim.errors.any? %>
 
     <%= form_for current_claim, url: claim_path do |form| %>
       <%= form_group_tag current_claim do %>
+        <%= form.fields_for :eligibility, include_id: false do |fields| %>
         <h1 class="govuk-label-wrapper">
```

Comment:
Indentation nitpick: shouldn't this all be indented another level now?

Reply:
👍🏽

Reply:
👍